frontend/workflows/dynamodb/src/update-capacity.tsx

```diff
@@ -157,11 +162,16 @@ const TableDetails: React.FC<TableDetailsChild> = ({ enableOverride }) => {
 
       {enableOverride && (
         <Box>
-          <Alert severity="warning">
-            Warning: to override the DynamoDB scaling limits, check the box below. This will bypass
-            the maximum limits placed on all throughput updates. Only override limits if safe to do
-            so.
-          </Alert>
+          {limitsNotes.length === 0 && (
+            <Alert severity="warning">
+              Warning: to override the DynamoDB scaling limits, check the box below. This will
+              bypass the maximum limits placed on all throughput updates. Only override limits if
+              safe to do so.
+            </Alert>
+          )}
+
+          <NotePanel notes={limitsNotes} />
```

Comment:
Small nit just so we dont render anything if there aren't any notes
```suggestion
          {limitsNotes.length > 0 && <NotePanel notes={limitsNotes} />}
```